src/editor/panel.ts

```diff
@@ -70,15 +70,15 @@ export class EditorPanel extends Panel {
   }
 
   onUpdateRequest(): void {
-    this._gridstackWidget.update();
+    this._gridstackWidget?.update();
   }
 
   get gridWidgets(): Widget[] {
-    return this._gridstackWidget.gridWidgets;
+    return this._gridstackWidget!.gridWidgets;
```

Comment:
Wondering whether this could be written like this instead?

```suggestion
    return this._gridstackWidget?.gridWidgets ?? [];
```

Also it looks like the `strictNullChecks` forced the use of `!` in many places. Maybe there is something we can do to limit its use? (since it's also an eslint warning)


It would be fine to address in a separate cleanup PR.

Reply:
Thanks, Jeremy. That sounds good to me. I'll change in a new PR.